src/lib/UriActions/generate.ts

```diff
@@ -0,0 +1,66 @@
+import axios from 'axios';
+import { UriAction, UriActionType } from 'types/uri';
+import { err, Result, ResultAsync } from 'neverthrow';
+import { actionUriFromRecord } from 'lib/UriActions/parsing';
+import { fromBase64 } from '@cosmjs/encoding';
+
+const axiosInstance = axios.create({
+  baseURL: 'http://57.128.144.235:33000/deep-links',
```

Comment:
We should probably move this Axios instance in another file since in the future we will use it for other APIs calls (i.e. registering the notification token of the user).

I will also wait to merge this PR until we have a domain name for these APIs

Reply:
@RiccardoM I moved the axios instance. Additionally, I've excluded `/deep-links` from the `baseURL` to ensure compatibility with upcoming actions.